### PR TITLE
Refactor & SSL DCV

### PR DESCRIPTION
**Pull request by tlianza**
- Simplified response structs by using embedded types.
- Added the endpoint for getting ssl DCV records from the API.

### CODE REVIEW

**Comment by terinjokes** [comment]
@tlianza sad failure from not running `gofmt`. 🐼

**Comment by tlianza** [comment]
@terinjokes fixed!

File: ssl.go [hunk 1]
```diff
@@ -42,3 +47,19 @@ func (c *API) ReprioSSL() {
 // 	DELETE /zones/:zone_identifier/custom_certificates/:identifier
 func (c *API) DeleteSSL() {
 }
+
+// SSLVerification gets the Domain Control Validation records customers
+// need to validate ownership of a domain for Universal SSL
+//  GET /zones/:zone_identifier/ssl/verification"
+func (c *API) SSLVerification(zoneID string) (SSLVerificationResponse, error) {
```
Comment (jamesog) [inline ssl.go]: Please make the method receiver signature `api *API` - we're making this consistent everywhere now. Early attempts were a little mixed. :-)

File: cloudflare.go [hunk 2]
```diff
@@ -444,3 +444,20 @@ type IPsResponse struct {
 	Response
 	Result IPRanges `json:"result"`
 }
+
+type SSLVerification struct {
```
Comment (jamesog) [inline cloudflare.go]: Please add a description comment to each type. It should be of the format:

```
// SSLVerification (is|describes|etc) ...
```

File: ssl.go [hunk 3]
```diff
@@ -42,3 +47,19 @@ func (c *API) ReprioSSL() {
 // 	DELETE /zones/:zone_identifier/custom_certificates/:identifier
 func (c *API) DeleteSSL() {
 }
+
+// SSLVerification gets the Domain Control Validation records customers
+// need to validate ownership of a domain for Universal SSL
+//  GET /zones/:zone_identifier/ssl/verification"
```
Comment (jamesog) [inline ssl.go]: Nit-pick: trailing "

File: cloudflare.go [hunk 4]
```diff
@@ -444,3 +444,20 @@ type IPsResponse struct {
...
+	VerificationType  string              `json:"verification_type"`
+	VerificationInfo  SSLVerificationInfo `json:"verification_info,omitempty"`
+}
+
+type SSLVerificationInfo struct {
+	RecordName   string `json:"record_name"`
+	RecordTarget string `json:"record_target"`
+}
+
+type SSLVerificationResponse struct {
```
Comment (elithrar) [inline cloudflare.go]: Make this private (as per #78)

**Comment by elithrar** [comment]
Just needs those few comments closed out.

**Pull request closed without merging**